charts/istio/istio-ingress/templates/autoscale.yaml

```
@@ -16,4 +16,10 @@ spec:
   - type: Resource
     resource:
       name: cpu
+      {{- if semverCompare "<= 1.22.x" .Capabilities.KubeVersion.GitVersion }}
```

Shouldn't the comparison be

```suggestion
      {{- if semverCompare "<= 1.22-0" .Capabilities.KubeVersion.GitVersion }}
```

I recall in the past the `1.22.x` way did not work when Seed is a non Gardener managed cluster and contains a custom version suffix (`v1.19.12-some-suffix`). See https://github.com/gardener/gardener/pull/4676